### PR TITLE
Add info about single module recompilation ##doc

### PR DESCRIPTION
**Pull request by satk0**
- [x] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -564,6 +564,14 @@ export CC="ccache gcc"
 This will automatically detect when files do not need to recompiled and avoid
 unnecessary work.
 
+Huge increase in recompilation speed could also be achieved by recompilating only modified modules (directories) by using:
```
Comment (trufae) [inline DEVELOPERS.md:567]:
```suggestion
it is not necessary to recompile the whole project everytime. Just run make in the directory you are working on to compile these parts only by using:
```

**Review by trufae — CHANGES_REQUESTED** [review]
What about sthg like this?

**Comment by trufae** [comment]
yeah, also mention that symlink thing :) it will be clearer. thanks

**Comment by satk0** [comment]
Done. Let me know how it looks

**Pull request merged**